### PR TITLE
qa,src: update mypy to 0.782

### PR DESCRIPTION
**Pull request by sebastian-philipp**
Signed-off-by: Sebastian Wagner <sebastian.wagner@suse.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: src/pybind/mgr/dashboard/rest_client.py [hunk 1]
```diff
@@ -511,7 +511,7 @@ def func_wrapper(self, *args, **kwargs):
                 resp_structure = api_kwargs.get('resp_structure', None)
                 args_name = inspect.getargspec(func).args
                 args_dict = dict(zip(args_name[1:], args))
-                for key, val in kwargs:
+                for key, val in kwargs.items():
```
Comment (sebastian-philipp) [inline src/pybind/mgr/dashboard/rest_client.py:514]: @votdev + @p-se : is this code actually executed?

Comment (alfonsomthd) [reply src/pybind/mgr/dashboard/rest_client.py:514]: @sebastian-philipp This seems a hidden bug that we should backport...

**Comment by tchaikov** [comment]
https://pulpito.ceph.com/kchai-2020-06-28_05:13:09-rados-wip-kefu2-testing-2020-06-28-1028-distro-basic-smithi/

**Comment by sebastian-philipp** [comment]
Thank you kefu.

**Pull request merged**